test/modules/bradc/printModStuff/foo.good

```diff
@@ -49,6 +49,9 @@ end of module search dirs
   $CHPL_HOME/modules/layouts/LayoutCS.chpl
   $CHPL_HOME/modules/dists/SparseBlockDist.chpl
   $CHPL_HOME/modules/dists/ReplicatedDist.chpl
+  $CHPL_HOME/modules/standard/Memory.chpl
+  $CHPL_HOME/modules/standard/Memory/Diagnostics.chpl
+  $CHPL_HOME/modules/standard/Memory/Initialization.chpl
```

Comment:
I didn't think we were using Set in the included by default modules but I guess we are. Do you happen to know where?

Reply:
I'm not sure where. @lydia-duncan mentioned something about it earlier?

Reply:
It's not included by default but I believe it is still used in a default included module.  There's a Sung test that prints the module initialization order, I think that'll indicate where it is first used

Reply:
Looks like it is used in `standard/Random.chpl`, in the implementation of `_choiceUniform`.